probes/http/http.go

```diff
@@ -162,6 +165,7 @@ func (p *Probe) getTransport() (*http.Transport, error) {
 	if p.c.GetDisableHttp2() {
 		// HTTP/2 is enabled by default if server supports it. Setting
 		// TLSNextProto to an empty dict is the only way to disable it.
+		// This only works if transport hasn't been previously cloned. See https://github.com/cloudprober/cloudprober/issues/872
```

Comment:
One minor nit re: comment line length :)

```suggestion
		// This only works if transport hasn't been previously cloned. See
		// https://github.com/cloudprober/cloudprober/issues/872
```